lib/api.common.js

```diff
@@ -594,18 +594,17 @@ api.common.sortCompareByName = (
   return 0;
 };
 
-api.common.clearCacheStartingWith = (
+api.common.clearCacheStartsWith = (
```

Comment:
Ugh, never mind. For some reason I only reviewed the first commit, not the whole PR.